core/state/statedb.go

```diff
@@ -85,6 +85,34 @@ func (s *StateObjectSyncMap) StoreStateObject(addr common.Address, stateObject *
 	s.Store(addr, stateObject)
 }
 
+// loadStateObjectFromStateDB is the entry for loading state object from stateObjects in StateDB or stateObjects in parallel
+func (s *StateDB) loadStateObjectFromStateDB(addr common.Address) (*StateObject, bool) {
+	if s.isParallel {
+		return s.parallel.stateObjects.LoadStateObject(addr)
```

Comment:
s.stateObjects && s.parallel.stateObjects could be combined, with a common Interface? so we can remove  s.parallel.stateObjects

type StateObjects interface {
	LoadStateObject() *StateObject
    ...
}

Reply:
no help, ignore